mathlibtools/import_graph.py

```diff
@@ -66,3 +66,10 @@ def path(self, start: str, end: str) -> 'ImportGraph':
         H = self.subgraph(set(D.nodes).intersection(A.nodes))
         H.base_path = self.base_path
         return H
+
+    def exclude_tactics(self) -> 'ImportGraph':
+        """Removes all files in src/tactic/ and src/meta/ from the graph."""
+        H = self.subgraph([n for n,a in self.nodes(data=True) if
+          not a['label'].startswith('tactic') and not a['label'].startswith('meta')])
```

Comment:
Maybe
```suggestion
          not a['label'].startswith('tactic'.) and not a['label'].startswith('meta.')])
```
so as not to rule out third party names like `metamathematics`. Is `.` the separator or `/`?

Reply:
`.` is the separator: that's what displays when you view the graphs.